website/docs/docs/get-started/develop-in-the-cloud.md

```diff
@@ -19,9 +19,9 @@ With the Cloud IDE, you can:
     * Compile Python models to see the full function that gets executed in your data platform
     * See Python models in DAG in dbt version 1.3 and higher
     * Currently, you can't preview python models
-- Visualize a directed acyclic graph (DAG), and more.
+- Visualize a directed acyclic graph (DAG), and [more](#cloud-ide-features).
 
-<Lightbox src src="/img/docs/dbt-cloud/cloud-ide/cloud-ide-v2.jpg" title="The dbt Cloud IDE in dark mode"/>
+<Lightbox src src="/img/docs/dbt-cloud/cloud-ide/cloud-ide-v2.jpg" width="85%" title="The dbt Cloud IDE with all its features is visible in dark mode."/>
 
 
 ## Prerequisites
```

Comment:
hey @bossnunta - one of the prerequisites is dbt version 0.15.0 or higher (see below), is this now incorrect due to the <1.0 deprecation?

>Your dbt project must be compatible with dbt version 0.15.0 or higher. The dbt IDE is powered by the [dbt-rpc](https://github.com/dbt-labs/docs.getdbt.com/blob/31ce8f4e4ec01b82d7e6c5aff8b1af1befe832de/reference/commands/rpc) which was overhauled in dbt v0.15.0